src/main/java/edu/hawaii/its/groupings/service/OotbActiveUserProfileService.java

```diff
@@ -0,0 +1,60 @@
+package edu.hawaii.its.groupings.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.security.core.authority.AuthorityUtils;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+import edu.hawaii.its.groupings.access.User;
+
+public class OotbActiveUserProfileService implements UserDetailsService {
+    private final Map<String, User> users = new HashMap<>();
+
+    public OotbActiveUserProfileService() {
+        initUsers();
+    }
+
+    private void initUsers() {
+        // Initialize user with UID "user0123"
+        users.put("USER", new User.Builder("user0123")  // UID is clearly the first parameter in the constructor
```

Comment:
Make sure to rename `USER` to `MEMBER`, as Michael mentioned in our last meeting.

Reply:
Thanks for reminding me